paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/reporting/ScreenshotClassPageRenderer.kt

```diff
@@ -0,0 +1,147 @@
+package app.cash.paparazzi.gradle.reporting
+
+import com.android.build.gradle.internal.test.report.ErroringAction
+import com.android.build.gradle.internal.test.report.SimpleHtmlWriter
```

Comment:
We should probably avoid relying on internal classes, any alternatives?

Reply:
what about `org.gradle.internal.html.SimpleHtmlWriter` instead?

Reply:
I updated the pr with a couple more internal classes to remove these usages